wiki/deployment/deploying-your-application.md

```diff
@@ -11,11 +11,11 @@ The mechanism to deploy your game differs between platforms. This article aims t
 # Deploy to Windows/Linux/Mac OS X
 ### As JAR file
 The easiest way to deploy to Windows/Linux/Mac is to create a runnable JAR file. This can be done via the following console command:
-`./gradlew desktop:dist`
+`./gradlew lwjgl3:dist`
 
 If you are getting an error like `Unsupported class file major version 60`, your Java version (see [here](https://stackoverflow.com/q/9170832) for a list) is not supported by your Gradle version. To fix this, install an older JDK.
 
-The generated JAR file will be located in the `desktop/build/libs/` folder. It contains all necessary code as well as all your art assets from the android/assets folder and can be run either by double clicking or on the command line via `java -jar jar-file-name.jar`. Your audience must have a JVM installed for this to work. The JAR will work on Windows, Linux and Mac OS X!
+The generated JAR file will be located in the `lwjgl3/build/libs/` folder. It contains all necessary code as well as all your art assets from the android/assets folder and can be run either by double clicking or on the command line via `java -jar jar-file-name.jar`. Your audience must have a JVM installed for this to work. The JAR will work on Windows, Linux and Mac OS X!
```

Comment:
> The JAR will work on Windows, Linux and Mac OS X!

This should be changed to `macOS`. In addition, it's not really true because of the whole XstartOnFirstThread stuff. IMO we should move the disclaimer that was previously in other place right here:

```md
On **macOS**, there is another step required to get your JAR file running. You need to make sure that your application’s `main()` method runs on the first (i.e., the AppKit) thread. This is done by setting the `-XstartOnFirstThread` for your JVM. In your development environment, when you're starting your project via Gradle, this is already taken care of by the run configuration. For JAR files, this is best done by either adding the ([experimental](https://github.com/libgdx/libgdx/issues?q=is%3Aissue+is%3Aopen+label%3Aglfw-awt-macos)) `com.badlogicgames.gdx:gdx-lwjgl3-glfw-awt-macos` dependency to your desktop project or by programatically restarting the JVM if the required `-XstartOnFirstThread` argument is not present (see [here](https://github.com/crykn/guacamole/blob/master/gdx-desktop/src/main/java/de/damios/guacamole/gdx/StartOnFirstThreadHelper.java#L77) for a simple example). If you are packaging a JRE with your game (which is the recommended way to distribute your later game), the tool of your choice typically allows you to set the JVM arguments.
```

Reply:
StartupHelper is included and called by default in all lwjgl3 projects, are you saying it isn't enough on macOS?

Reply:
Ah, I wasn't aware of that. Consider this resolved then!